docs/change-log/index.md

```diff
@@ -13,6 +13,52 @@
 - [Deprecations and removed code](#deprecations-and-removed-code)
  
 
+## v1.6.3 (4 June 2024)
+
+### Workflows
+| ID       |Description                                                               |
+|----------|---------------------------------------------------------------------------|
+|ML-3521,5482|Remote/scheduled workflows can now be performed by a project with a source that is contained on the image. See [Scheduling a workflow](../concepts/scheduled-jobs.html#scheduling-a-workflow). Tech Preview. |
+
+### Infrastructure
+
+| ID    |Description                                                                 |
+|-------|----------------------------------------------------------------------------|
+|ML-5739|MLRun now supports email-like username.|
+
+### Documentation
+| ID    |Description                                                               |
+|-------|---------------------------------------------------------------------------|
+|ML-4620| Updated [Realtime monitoring and drift detection tutorial](../tutorials/05-model-monitoring.html) and {ref}`monitoring-overview` for the [model monitoring](#model-monitoring) feature introduced in v1.6.0.|
+|NA     | New page describing {ref}`log-artifacts`.                                 |
+|NA     | New page describing {ref}`parallel-workflows`.                                 |
+|NA     | New page describing {ref}`conditional-workflow`.                                 |
+|NA     | New page describing {ref}`exithandler-workflow`.                                 |
+
+### Breaking change
+| ID          |Description                                                               |
+|-------|---------------------------------------------------------------------------|
+|ML-6098|The `prediction` and `named_predictions` columns (list of all predictions) were removed from the model monitoring parquet files. Each prediction is still available in a column of its own.|
+ML-6397|`list runs`  API does not return artifacts in the run body. Instead you get a dictionary of artifact_uris with path to the artifacts in the DB. You can get the artifacts only if you use the `get run`API (i.e. when getting a single run). If an artifact is deleted, it will not be returned in the run body.|
```

Comment:
```suggestion
ML-6397|`list runs`  API does not return artifacts in the run body. Instead you get a dictionary of artifact_uris with store path to the artifacts which can be retrieved via `mlrun.store_manager`. You can get the artifacts in the run status, only if you use the `get run`API (i.e. when getting a single run). If an artifact is deleted, it will not be returned in the run body.|
```